docs/querying/operators.md

```diff
@@ -218,6 +218,10 @@ number of seen HTTP requests per application and group over all instances via:
 
     sum(http_requests_total) without (instance)
 
+Which is equivalent to:
+
+    sum(http_requests_total) by (application,group)
```

Comment:
Space after comma

Reply:
updated.

Feel free to change anything else that doesn't look right.